cachito/workers/nexus.py

```diff
@@ -14,6 +14,43 @@
 log = logging.getLogger(__name__)
 
 
+def _get_nexus_hoster_credentials():
+    """
+    Get the username and password of the account to use on Nexus instance that hosts content.
+
+    :return: a tuple of username and password
+    :rtype: tuple(str, str)
+    """
+    config = get_worker_config()
+
+    if config.cachito_nexus_hoster_username:
+        username = config.cachito_nexus_hoster_username
```

Comment:
Should we require the "hoster" password if the "hoster" username is being used? This may cause confusion otherwise.